client/src/pages/index.tsx

```diff
@@ -274,18 +279,30 @@ function IndexPage(props: {
             </div>
             <div className={styles.flexExpandChild}>
               {mentorType === MentorType.CHAT ? (
-                <Chat height={chatHeight} />
+                <Chat
+                  height={chatHeight}
+                  chatProps={{
```

Comment:
please, no nested `chatProps`

Reply:
How can I pass the `chatProps` without nested props?

Reply:
any functional component in react should take at most a single argument `props` all props for that component are defined there.

```typescript
export function Chat(props: {
  height?: number,
  displayMentorNames?: string;
  //...
}) {
// ...
}
```

```tsx
<Chat height="1000" displayMentorNames={true}/>
```

Reply:
done